parser.y

```diff
@@ -484,15 +482,17 @@ import (
 	varPop			"VAR_POP"
 	varSamp			"VAR_SAMP"
 
-	/* The following tokens belong to TiDBKeyword. */
+	/* The following tokens belong to TiDBKeyword. Notice: make sure these tokens are contained in TiDBKeyword. */
 	admin		"ADMIN"
 	buckets		"BUCKETS"
 	cancel		"CANCEL"
-	ddl		"DDL"
-	drainer         "DRAINER"
+	ddl		    "DDL"
+	drainer     "DRAINER"
 	jobs		"JOBS"
 	job		    "JOB"
-	pump            "PUMP"
+	node_id		"NODE_ID"
```

Comment:
```suggestion
	nodeId		"NODE_ID"
```
It's better to keep consistent with others.

Reply:
I think `nodeID` is better.